explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts

```diff
@@ -153,23 +155,26 @@ export const samplesBuildSampleType = (
 };
 export const samplesBuildLibConsApproach = (
   sample: SamplesResponse
-): React.ComponentProps<typeof C.Cell> => {
+): React.ComponentProps<typeof C.NTagCell> => {
   return {
-    value: Transformers.samplesGetLibConsApproach(sample),
+    label: getPluralizedMetadataLabel(METADATA_KEY.LIB_CONST_APPROACH),
+    values: Transformers.samplesGetLibConsApproach(sample),
```

Comment:
For clarity, let's update here and elsewhere any methods specific to library construction approach to the full name i.e. `Transformers.samplesGetLibConsApproach` becomes `Transformers.samplesGetLibraryConstructionApproach`.

Reply:
While we are updating the method names we might as well refactor `samplesBuildLibConsApproach` to `samplesBuildLibraryConstructionApproach`. Would you mind doing this for me here and elsewhere? Thank you!